### PR TITLE
General updates

### PR DESCRIPTION
**Pull request by ryangribble**
- Update README and shipping-releases info as they are a bit out of date
- Update year/copyright info on assemblies
- Update cake.frosting package to latest version for better dotnet msbuild tooling support
- Update xunit packages to latest version (fixes CodeLens display of nested test classes in VS2017 - also requires VS2017 15.3 update)

### CODE REVIEW

**Comment by ryangribble** [comment]
@mderriey could you have a sanity 👀 before I merge this?

File: docs/shipping-releases.md [hunk 1]
```diff
@@ -2,43 +2,32 @@
 
 ## Pre-requisites
 
- - Visual Studio 2013 Update 4
+ - Visual Studio 2017 Update 4
```
Comment (M-Zuber) [inline docs/shipping-releases.md]: Isn't this 2017 update 3?

**Comment by M-Zuber** [comment]
Other than that one comment, it LGTM

File: docs/shipping-releases.md (hunk 1 above)
Comment (ryangribble) [inline docs/shipping-releases.md]: Ah whoops I think I just did a replace all 2014 -> 2017... thanks!

**Pull request merged**